newsroom/web/default_settings.py

```diff
@@ -375,3 +375,5 @@
 DATE_FORMAT_HEADER = "EEEE, dd.MM.yyyy"
 DATETIME_FORMAT_SHORT = "short"
 DATETIME_FORMAT_LONG = "dd/MM/yyyy HH:mm"
+
+PREPEND_EMBARGOED_TO_WIRE_SEARCH = strtobool(env('PREPEND_EMBARGOED_TO_WIRE_SEARCH', 'false'))
```

Comment:
wouldn't probably set this via environment, so setting it to just `False` by default would be fine imo